lib/modules/datasource/flutter-version/types.ts

```diff
@@ -0,0 +1,22 @@
+export interface FlutterCurrentRelease {
+  beta: string;
+  dev: string;
+  stable: string;
+}
+
+export interface FlutterRelease {
+  hash: string;
+  channel: string;
```

Comment:
```suggestion
  channel: 'stable' | 'beta' | 'dev';
```
small improvement for stronger typing 😉